rsts/community/troubleshoot.rst

```diff
@@ -3,13 +3,13 @@
 Troubleshooting Guide
 ---------------------
 
-.. admonition:: Why have we crafted this guide?
+.. admonition:: 
```

Comment:
I think you ought to have a title for admonition.

Reply:
I don't see "admonition" on the preview page. Can we safely delete it?

Reply:
The line below "admonition" has to have a tab before the line starts. 
It's totally up to you whether to delete or retain it.

Reply:
@cosmicBboy What do you think?